bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/Tree.java

```diff
@@ -5371,7 +5371,7 @@ public void setTopItem (TreeItem item) {
  * In a Tree without imageList, the indent also controls the chevron (glyph) size.
  */
 private void calculateAndApplyIndentSize() {
-	int indent = DPIUtil.autoScaleUpUsingNativeDPI(DEFAULT_INDENT);
+	int indent = DPIUtil.scaleUp(DEFAULT_INDENT, nativeZoom);
```

Comment:
Isn't this "conflicting" with #1330? Maybe we should have a separate PR for the replacement of the `DPIUtil.autoScaleUpUsingNativeDPI()` calls, in particular because I think that method is completely unused after the changes in this PR (and makes no sense anymore anyway), so it should probably be removed?

Reply:
I am not sure if it is used for some other platform though. I can have a look.

Reply:
Alright its just used for win32